react-client/README-DEV.md

```
@@ -0,0 +1,74 @@
+## React API Client: Generated from OpenAPI
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)